helm/nodejs-ex/templates/buildconfig.yaml

```diff
@@ -0,0 +1,25 @@
+kind: BuildConfig
+apiVersion: build.openshift.io/v1
+metadata:
+  name: nodejs-ex
+  labels:
+    app: nodejs-ex
+spec:
+  strategy:
+    type: Source
+    sourceStrategy:
+      from:
+        kind: ImageStreamTag
+        name: nodejs:6
```

Comment:
not going to parameterize any of this?

Reply:
updated - this file is now a literal translation of openshift/templates/nodejs.json with the same parameterisation.